test-case-reporting/app.ts

```diff
@@ -142,12 +143,19 @@ app.post('/report', jsonParser, async (req, res) => {
 });
 
 app.get('/_cron/compute-stats', async (req, res) => {
+  const testCaseStats = new TestCaseStats(db);
   const {count} = req.query;
 
-  const message = `Computing pass/fail % for past ${count} runs`;
-  res.send(message);
+  try {
+    const sampleSize = parseInt(count.toString());
+    await testCaseStats.updateStats(sampleSize);
 
-  console.log(message);
+    const message = `Computed pass/fail % for past ${count} runs`;
+    res.send(message);
```

Comment:
This is a cron job, so there is no receiving browser. Let's `.sendStatus(statusCodes.OK)` on success and similarly send just an error status on failure, but still log the message or error for debugging purposes.

Reply:
Done.